system/t04_mirror/UpdateMirror11Test_gold

```diff
@@ -13,9 +13,6 @@ Downloading ftp://ftp.ru.debian.org/debian/pool/main/s/sensible-utils/sensible-u
 Downloading ftp://ftp.ru.debian.org/debian/pool/main/s/sysvinit/sysvinit-utils_2.88dsf-59.9_i386.deb...
 Mirror `stretch-main` has been successfully updated.
 Packages filtered: 50621 -> 3.
-gpgv: Good signature from "Debian Archive Automatic Signing Key (7.0/wheezy) <ftpmaster@debian.org>"
```

Comment:
these and the two below should be removed from the PR, will get back once tests are stable enough